net_udp.go

```diff
@@ -64,5 +60,5 @@ func newNetUDPSummary(file string) (*NetUDPSummary, error) {
 		return nil, err
 	}
 	n1 := NetUDPSummary(*n)
-	return &n1, err
+	return &n1, nil
```

Comment:
I think this should probably stay `return &n1, err`, in case `newNetIPSocketSummary()` returns a non-nil result and an error.